src/redux/reducers/index.js

```diff
@@ -1,18 +1,23 @@
 import { combineReducers } from 'redux';
```

Comment:
I don't think we should use the index to have any "cases" in it. The index should only contains all the imported reducers and combined them. We should create a reducer file for the different components, and then imported them to the index. How everyone feel about it?